sparse_ho/models/lasso.py

```diff
@@ -368,8 +369,8 @@ def get_jac_v(self, X, y, mask, dense, jac, v):
         return jac.T @ v(mask, dense)
 
     @staticmethod
-    def get_hessian(X, y, mask, dense, log_alpha):
-        """Compute Hessian of datafit.
+    def get_mv(X, y, mask, dense, log_alpha):
+        """Compute matrix vector product with the Hessian of datafit.
```

Comment:
In fact it does not return a product, it returns a LinearOperator

Reply:
I changed the docstring